src/components/QuizPage/QuizPage.tsx

```diff
@@ -20,42 +20,54 @@ interface IQuizPageProps {
   name: string
 }
 
-
 const QuizPage: React.FC<IQuizPageProps> = ({ currentCountry }) => {
-  const [questions, setQuestions] = useState< string | null > (null)
+  const [questions, setQuestions] = useState< string>('')
   const [country, setCountry] = useState<any>()
   const questionList = [
     'What is the population of',
     'What is the capital of',
     'How many countries border'
   ]
-  
-   // ********* Randomizer ********//
 
+  // ***********************************************
+      /*UPDATING STATE, GETTING LOCAL STORAGE*/ 
+  // ***********************************************
   useEffect(() => {
     if (currentCountry && currentCountry.name) {
-      setCountry(currentCountry)
-      getRandomElement(currentCountry.name)
+      setCountry(currentCountry);
+      getRandomElement(currentCountry.name);
     } else {
-      setCountry(JSON.parse(localStorage.getItem('currentCountry')!))
-      getRandomElement(JSON.parse(localStorage.getItem('currentCountry')!).name)
+      setCountry(JSON.parse(localStorage.getItem('currentCountry')!));
+      getRandomElement(JSON.parse(localStorage.getItem('currentCountry')!).name);
     }
   }, [])
 
+  // ***********************************************
+          /*RANDOMIZING QUIZ QUESTIONS*/ 
+  // ***********************************************
   const getRandomElement = (name: string) => {
-  const result = questionList[Math.floor(Math.random() * questionList.length)] + ' ' + name
-   setQuestions((result))
+  const result = questionList[Math.floor(Math.random() * questionList.length)] +` ${name}`
+   setQuestions((result));
   }
   
-  
   return (
     <div className="display-area quiz-display" >
       <section className="display-area" >
         {currentCountry && (
         <div className="quiz-contents">
-          <img src={currentCountry.flag} alt={`${currentCountry.name} flag`} className="flag-img" />
-         {country  && <h2>{questions}?</h2>}
-        {country  && <Form questions={`${questions} ${country.name}?`} currentCountry={country}/>}  
+          {country && 
```

Comment:
Good call adding the conditional to keep the flag displayed. A great catch to make the app less buggy